src/dao/ContractsManagerDAO.js

```diff
@@ -54,9 +54,9 @@ const daoMap = {
   [DAO_EXCHANGE_MANAGER]: ExchangeManagerDAO,
   [DAO_EXCHANGE]: ExchangeDAO,
   [DAO_ERC20_MANAGER]: ERC20ManagerDAO,
-  [DAO_VOTING]: VotingDAO,
-  [DAO_VOTING_DETAILS]: VotingDetailsDAO,
-  [DAO_VOTING_ACTOR]: VotingActorDAO,
+  [DAO_VOTING_MANAGER]: VotingManagerDAO,
```

Comment:
actually contractManager returns only ~10 addresses, but we have about ~20 types. Ask @ahiatsevich how to sync this types this our code. May be update contractManager and add method `getContracts` with `(address[], names[])` return.